File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = [0] * len(prices)
    for i in range(len(prices)-1):
        price = prices[i]
        for j in range(i+1, len(prices)):
            answer[i] += 1
            if (price >  prices[j]):
                break
                
    return answer

File: 프로그래머스/2/42584. 주식가격/주식가격.java
class Solution {
    public int[] solution(int[] prices) {
        int[] answer = new int[prices.length];
        
        for (int i=0; i<prices.length-1; i++) {
            Integer currnetPrice = prices[i];
                for (int j=i+1; j<prices.length; j++) {
                    answer[i] += 1;
                    if (currnetPrice > prices[j]) {
                        break;
                    }
                }
        }
        return answer;
    }
}